test/e2e-common.sh

```diff
@@ -365,36 +347,6 @@ function ingress_class() {
   fi
 }
 
-# Uninstalls Knative Serving from the current cluster.
-function knative_teardown() {
```

Comment:
Is it intended to remove this function?

Reply:
I removed it because the callsite wasn't obvious (although I question whether practically anybody uses the flow that hits this?).  I have added it back in #6434 , but effectively rewritten it because it didn't handle non-Istio at all.